tests/benchmarks/multicore/idle_wdt/remote/boards/nrf54h20dk_nrf54h20_cpurad.overlay

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/ {
+	aliases {
+		watchdog0 = &wdt010;
+	};
+};
+
+&wdt010 {
+	status = "okay";
+};
+
+&rng_hci {
+	status = "disabled";
```

Comment:
Same like for cpuapp. Added in hope to lower current consumption. However, no impact was observed.
Shall I remove this?

Reply:
removed